docs/user_workloads/instancetypes.md

```diff
@@ -328,7 +331,10 @@ $ kubectl patch VirtualMachineInstancetype/csmall --type merge -p '{"spec":{"cpu
 virtualmachineinstancetype.instancetype.kubevirt.io/csmall patched
 ```
 
-In order for this change to be picked up within the `VirtualMachine`, we need to stop the running `VirtualMachine` and clear the `revisionName` referenced by the `InstancetypeMatcher`:
+!!! Note
```

Comment:
Not correct, the valid way to pickup changes to the contents of an instancetype/preference is to clear the `revisionName` (i.e. what was said before).